static/i18n/es.js

```diff
@@ -1,3 +1,3 @@
 export default {
-  "name" : "Okuna"
+  "name" : "Coac"
```

Comment:
```suggestion
  "name" : "COAC"
```

Reply:
¿could be a deleted this name?. it shows on the login page and I don't believe  that necessary